src/util/lie/se3.cpp

```diff
@@ -0,0 +1,34 @@
+#include "se3.hpp"
+
+#include <utility>
+
+SE3::SE3(Eigen::Vector3d position, SO3 rotation) : mPosition(std::move(position)), mRotation(std::move(rotation)) {
```

Comment:
Relying on an implicit move ctor for SO3 here. Please mark it as default in the class definition. Maybe delete copy for good measure too.

Reply:
Why delete copy? I think that is a fine operation.

Reply:
yeah thats fine